pkg/executor/internal/testutil/testutil.go

```diff
@@ -144,6 +148,9 @@ func (*MockDataSource) RandDatum(typ *types.FieldType) interface{} {
 
 // PrepareChunks prepares chunks
 func (mds *MockDataSource) PrepareChunks() {
+	if mds.P.SaveChunks {
+		mds.SavedChunks = make([]*chunk.Chunk, len(mds.GenData))
+	}
```

Comment:
why add this?

Reply:
> why add this?

Because we need to collect the chunks that are output by this mock source so that we can calculate expected result to check the correctness of sort executor.